javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -429,6 +432,57 @@ public Object slang_persistent() {
         }
     }
 
+    public static class Filter extends Base {
+        private final Predicate<Integer> ALL = i -> true;
+        private final Predicate<Integer> SOME = i -> (i & 1) == 1;
+        private final Predicate<Integer> NONE = i -> false;
+
+        @Benchmark
+        public void java_mutable(Blackhole bh) {
```

Comment:
testing `filter` is tricky: how many elements should it keep???
Therefore I added 3 variants: keeps everything, keeps around half and deletes everything.